src/workers/parser/utils/helpers.js

```diff
@@ -166,16 +166,20 @@ export function getVariables(dec: Node) {
       return [];
     }
 
-    // NOTE: it's possible that an element is empty
+    // NOTE: it's possible that an element is empty or has several variables
     // e.g. const [, a] = arr
-    return dec.id.elements.filter(element => element).map(element => {
-      return {
-        name: t.isAssignmentPattern(element)
-          ? element.left.name
-          : element.name || element.argument.name,
-        location: element.loc
-      };
-    });
+    // e.g. const [{a, b }] = 2
+    return dec.id.elements
```

Comment:
i kinda just want to remove variables because it populates a quick open view of variables that no one uses and i don't really agree with how it's done